flashbots/flashbots.py

```diff
@@ -90,9 +94,37 @@ def sign_bundle(
                 # sign the tx
                 signed_tx = signer.sign_transaction(tx)
                 signed_transactions.append(signed_tx.rawTransaction)
+            elif all(key in tx for key in ["v", "r", "s"]):
+                # transaction dict taken from w3.eth.get_block('pending', full_transactions=True)
+                v, r, s = (
+                    tx["v"],
+                    int(tx["r"].hex(), base=16),
+                    int(tx["s"].hex(), base=16),
+                )
+                raw = encode_transaction(
+                    Transaction(
+                        v=v,
+                        r=r,
+                        s=s,
+                        data=HexBytes(tx["input"]),
+                        gas=tx["gas"],
+                        gasPrice=tx["gasPrice"],
+                        nonce=tx["nonce"],
+                        to=HexBytes(tx["to"]),
```

Comment:
The `tx["to"]` field would be none during contract deployment if i understood it correctly. Should probably check for that first?

Reply:
Fixed